addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/ahamodel/TemperatureModel.java

```diff
@@ -31,15 +31,15 @@
 	private BigDecimal offset;
 
 	public BigDecimal getCelsius() {
-		return celsius.multiply(TEMP_FACTOR);
+		return celsius != null ? celsius.multiply(TEMP_FACTOR) : new BigDecimal("0");
 	}
 
 	public void setCelsius(BigDecimal celsius) {
 		this.celsius = celsius;
 	}
 
 	public BigDecimal getOffset() {
-		return offset.multiply(TEMP_FACTOR);
+		return offset != null ? offset.multiply(TEMP_FACTOR) : new BigDecimal("0");
```

Comment:
To avoid allocating an object unnecessarily, use [BigDecimal.ZERO](https://docs.oracle.com/javase/7/docs/api/java/math/BigDecimal.html#ZERO).